build.gradle.kts

```diff
@@ -250,3 +250,32 @@ license {
         set("company", "HRZN LTD")
     }
 }
+
+// inspired by https://github.com/TerraformersMC/GradleScripts/blob/2.0/ferry.gradle
+fun getVersionDecoration(): String {
+    if(project.hasProperty("release")) return ""
+
+    var version = "+build"
+    val branch = "git branch --show-current".execute()
+    if(branch.isNotEmpty() && branch != "main") {
```

Comment:
```suggestion
    if(branch.isNotEmpty() && branch.equals("main")) {
```

Reply:
wouldnt it be `!branch.equals("main")`?

Reply:
tho intellij seems to think that `!=` is better 
![image](https://user-images.githubusercontent.com/20533550/113523885-f768bb80-95fe-11eb-869b-6269eb410f9a.png)

Reply:
Ok, I assume kotlins equals handling is different than java then.